src/xtgeo/grid3d/_grid_roxapi.py

```diff
@@ -272,16 +271,13 @@ def _export_grid_cornerpoint_roxapi_v2(
     self: Grid, rox: RoxUtils, gname: str, realisation: int, info: bool
 ) -> None:
     """Convert xtgeo geometry to pillar spec in ROXAPI and store _xtgformat=2."""
-    try:
-        from roxar.grids import CornerPointGridGeometry as CPG
-    except ImportError:
-        raise RuntimeError("Cannot load Roxar module")
 
     grid_model = rox.project.grid_models.create(gname)
     grid_model.set_empty(realisation)
     grid = grid_model.get_grid(realisation)
 
-    geom = CPG.create(self.dimensions)
+    roxar_grids_: RoxarGridType = roxar_grids  # for mypy
+    geom = roxar_grids_.CornerPointGridGeometry.create(self.dimensions)
```

Comment:
```suggestion
    roxar_grids: RoxarGridType
    geom = roxar_grids.CornerPointGridGeometry.create(self.dimensions)
```
This might work? Just applies a type annotatation to the existing variable